plugins/liquid/refined-types-plugin/src/test/kotlin/arrow/meta/plugins/liquid/LiquidDataflowTests.kt

```diff
@@ -568,6 +568,45 @@ class LiquidDataflowTests {
       withoutPlugin = { compiles }
     )
   }
+
+  @Test
+  fun `class with two init blocks`() {
+    """
+      ${imports()}
+      data class A(val n: Int) {
+        init {
+          pre(n > 0) { "n must be positive" }
```

Comment:
How do we fail when multiple init blocks contain contradicting constraints? For example:

```kotlin
init {
  pre(x > 0) { "x > 0" }
}
init {
  pre(x < 0) { "x < 0" }
}
```

Reply:
This is handled by the inconsistency check for pre-conditions, in the same way that it's handled for the other functions (be aware that this is not yet implemented, but that's the idea).